e2e_test/batch/basic/table_with_default_columns.slt.part

```diff
@@ -44,7 +44,7 @@ select * from t2;
 2 2
 
 statement ok
-create table tt (v1 int, v2 timestamptz default now());
+create table tt (v1 int, v2 timestamptz default proctime());
```

Comment:
I'm not sure but IMO `now` is used with default column values (to follow the behavior of traditional databases) while `proctime` is for generated columns (to follow the behavior of streaming engines). So I suggest we may keep it untouched.

Though the support for `default now()` will be temporarily broken after #9762.

Reply:
We may utilize this to distinguish between `DEFAULT` and generated.

https://github.com/risingwavelabs/risingwave/blob/ca2109e65b24dd15edec9e6303ce8821b4a40f5b/src/frontend/src/binder/bind_context.rs#L73-L74

Reply:
done